### DISCUSSION TITLE
关于Waline在一个主题中部署后引起的动画问题！

### DISCUSSION THREAD

**Discussion by ghost** [walinejs/waline#629] [Q&A] [answered]
你好，我最近在适配 Hugo 的一个主题 [DoIt](https://github.com/HEIGE-PCloud/DoIt) ，发现 Waline 会与该主题有一些动画效果上的冲突。
我在与该主题的 Owner 沟通后，他给出的原因是：Waline 和 Valine 都直接向 header 中写入了自己的 CSS。其中的 `@keyframes pulse` 覆盖了 animate.css 中 pulse 的动画。

如果你对这个话题有兴趣，并且打算优化 Waline，请看问题详情：
请看：https://github.com/HEIGE-PCloud/DoIt/discussions/305

如果你没有兴趣，或者认为这不是一个应该通用的适配需求，请无视！
谢谢！

**Comment by lizheming** [comment, marked as answer]
`@waline/client@1.3.11` 版本已经去除了这些 keyframes 定义了，后期我们已经没用到这些动画了，你更新下再看看。

**Reply by ghost** [reply]
我是通过CDN引入waline的JS库，目前没有看到1.3.11的版本，还是1.3.10。

**Reply by Mister-Hope** [reply]
> 我是通过CDN引入waline的JS库，目前没有看到1.3.11的版本，还是1.3.10。

你需要清除cdn缓存
